evm_loader/program/src/entrypoint.rs

```diff
@@ -598,17 +598,31 @@ fn do_call<'a>(
         let backend = SolanaBackend::new(&account_storage, Some(accounts));
         debug_print!("  backend initialized");
 
-        let config = evm::Config::istanbul();
-        let mut executor = StackExecutor::new(&backend, usize::max_value(), &config);
+        // let config = evm::Config::istanbul();
```

Comment:
Please return original `do_call` implementation. We will move to our executor later.

Reply:
fixed